pkg/api/v1/clients/kubesecret/resource_client.go

```diff
@@ -23,12 +24,16 @@ import (
 
 const annotationKey = "resource_kind"
 
+var (
+	NotOurResource error = fmt.Errorf("not kube secret resource")
```

Comment:
make private, else lgtm

Reply:
i want this public to potentially be used from the SecretConverter interface